### DISCUSSION TITLE
Support request 6 days ago, no response (BLOCKING)

### DISCUSSION THREAD

**Discussion by MarcZijderveld** [community/community#52063] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Question

One of our colleagues has send a support ticket to Github Support 6 days ago (@BernardZijlstra CISO ) and this ticket has yet to be answered. The ticket is about his account being (incorrectly) flagged as a spam account and thus he is unable to work at all. After being a github user for 7+ years with a large organization we expected better support / respond times.

**Comment by rianrietveld** [comment]
I can confirm that @BernardZijlstra is not a spam account. Bernard is one of the partners at [Level Level](https://level-level.com/), and we need him to have access to GitHub.
